pasta_eln/database.py

```diff
@@ -836,6 +836,8 @@ def checkDB(self, outputStyle:str='text', repair:bool=False, minimal:bool=False)
           if not minimal:
             outstring+= outputString(outputStyle,'ok','..info: dataHierarchy exists')
           continue
+        if doc['_id'] == '-dataverseConfig-' or re.compile("^[a-f0-9]{32}$").match(doc['_id']):  #skip items that belong to dataverse
```

Comment:
The latest version creates all the dataverse related documents within a separate database named 'dataverse', so what is the need for this check? Do you look for documents in all the existing databases of couchdb?

Reply:
@SteffenBrinckmann Could you please see the comment added for this PR?